src/renderers/shared/fiber/__tests__/__snapshots__/ReactIncrementalPerf-test.js.snap

```diff
@@ -85,6 +85,8 @@ exports[`ReactDebugFiberPerf does not treat setState from cWM or cWRP as cascadi
   ⚛ (Committing Changes)
     ⚛ (Committing Host Effects: 2 Total)
     ⚛ (Calling Lifecycle Methods: 2 Total)
+
+⚛ (React Tree Reconciliation)
```

Comment:
`setState` inside `componentWillReceiveProps` schedules a callback if one is not already scheduled. We should add a 

This is an existing issue, though, not one introduced by this PR. It's only visible now because the new version of `ReactNoop.flush()` keeps flushing until there is no more work (until `scheduledDeferredCallback` is null), whereas before it would only flush the current callback.

The fix is simple but outside the scope of this PR.